src/core/actor_player.h

```diff
@@ -7,9 +7,8 @@ namespace core {
 
 class ActorPlayer : public Player {
  public:
-  ActorPlayer()
-      : Player(false) {
-  }
+  ActorPlayer() : Player(false) {}
+  virtual ~ActorPlayer() = default;
```

Comment:
can we discuss?

Reply:
Destructor has to be virtual because `ActorPlayer` is a polymorphic class https://cplusplus.com/doc/tutorial/polymorphism/ so it needs a virtual destructor (https://stackoverflow.com/a/461224).

Clang compiler warns if you don't do that.

Reply:
We might consider switching to Clang as a compiler, it seems to have way more warnings.

Reply:
Interesting